src/pudl/transform/ferc1.py

```diff
@@ -831,7 +831,8 @@ def transform_main(self, df: pd.DataFrame) -> pd.DataFrame:
             derived from the raw DBF and/or XBRL inputs.
         """
         df = (
-            self.normalize_strings(df)
+            self.spot_fix_values(df)
+            .pipe(self.normalize_strings)
```

Comment:
In the FERC1 context where do we think the spot fixes should be applied? There's some overlap between the kinds of things that can be fixed by spot fixes, and the string normalization / categorization etc.  The spot fixes feel like a fix of last resort, when the more general approaches can't work. Does it make sense for them to be the very first thing that's done? Do we foresee needing to apply different sets of spot fixes in more than one step?

Reply:
This is hard to answer and might end up being varying. There are some cases where you might want to spot fix things so that they get recognized and treated aptly by other functions and there are some cases where you might want to spot fix things after they have gone though all possible transforms as a final "buffing" of the data. 

Wherever we choose to put it there will undoubtedly be some tables where we'll have to override the transform and move the location of the spot fix function. I also think it's worth considering a scenario where we need to call spot fix at two different points of the transform process. I'm not sure our current setup can handle that.